### ISSUE TITLE
Right-clicking in Chromium DevTools doesn't work

### ISSUE THREAD

**Issue by NMinhNguyen** [HadoukenIO/core#110]
Hey,

I've noticed that right-clicking in DevTools in Openfin v6 doesn't seem to work. Was that a conscious decision, or a regression from v5?

**Comment by dcchuck** [comment]
The `--no-sandbox` flag will enable the rightclick in devtools.

**Comment by NMinhNguyen** [comment]
Are there any security implications when setting that flag?

**Comment by dcchuck** [comment]
Yes, you are disabling the chromium sandbox with that flag. Fine for testing, etc. but I wouldn't use in production.
